### PR TITLE
Fixed duplication of project version in children pom.xml

### PR DESCRIPTION
**Pull request by ZeyuLi-AWS**
*Issue #, if available:*

*Description of changes:*
* Removed explicit project version in children pom.xml 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,6 +33,7 @@
   </scm>
 
   <properties>
+    <revision>2.4.6-SNAPSHOT</revision>
```
Comment (stair-aws) [inline pom.xml:36]: Without context, `revision` is ambiguous. Suggest renaming to something more descriptive (e.g., `kcl.version`).

**Review by stair-aws — APPROVED** [review]
Verified "it blends", but recommend renaming the property to eliminate ambiguity.

@ZeyuLi-AWS Thanks for removing the duplication!

File: pom.xml (hunk 1 above)
Comment (ZeyuLi-AWS) [reply pom.xml:36]: In https://maven.apache.org/maven-ci-friendly.html#multi-module-setup, it mentioned: `A note about the used properties. You can only use those named ${revision}, ${sha1} and/or ${changelist} and not other named properties like this` 
Other names will be prohibited.

Comment (stair-aws) [reply pom.xml:36]: That limitation appears to be from `flatten-maven-plugin` [interpolation](https://github.com/mojohaus/flatten-maven-plugin/blob/c613968d1fe7b748945e4e232ecbe66739610feb/src/main/java/org/codehaus/mojo/flatten/cifriendly/CiModelInterpolator.java#L149-L153) -- a plugin we do not use today. Other property names appear to work.

LGTM either way!

**Pull request merged**